src/api.service.ts

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+
+import { AssetsResponse, GetAssetsParams } from './interfaces/assets.interface';
+
+export const BASE_URL = 'https://api.better-call.dev/v1';
+export const api = axios.create({ baseURL: BASE_URL });
+
+export async function requestGetAssets(params: GetAssetsParams): Promise<AssetsResponse> {
+  const { network, address, ...queryParams } = params;
+  const r = await api.get(`/account/${network}/${address}/token_balances`, {
+    params: queryParams
+  });
+  return r.data;
+}
```

Comment:
Let's use arrow functions everywhere.  
Do not use `async/await` (as we are using rxjs, all logic should be inside pipes). This function should return `api.get` directly.